src/core/util.ts

```diff
@@ -93,6 +93,10 @@ export function runBeforeNotifyHandlers(notice, handlers: BeforeNotifyHandler[])
 }
 
 export function runAfterNotifyHandlers(notice, handlers: AfterNotifyHandler[], error = undefined): boolean {
+  if (notice && notice.afterNotify) {
+    handlers.unshift(notice.afterNotify)
```

Comment:
Do you want to duplicate `handlers` before mutating it? I believe that's why we were doing the `Array.prototype.slice.call` dance. I don't know if it's necessary, though.

Reply:
I don't see why they were being duplicated though 🤷 .
One theory is that the array could be mutated from the initial call to `notify` until the `runAfterNotifyHandlers` is called, but that scenario is unlikely to happen.
So, I decided to use them in the same way we use the `runBeforeNotifyHandlers()`.
Should I clone them just to be on the safe side?

Reply:
Nope, I think it's fine as-is! I don't remember why they were duplicated either.